buildenv/jenkins/jobs/pipelines/Pipeline_Template.groovy

```diff
@@ -0,0 +1,106 @@
+if (!binding.hasVariable('DISCARDER_NUM_BUILDS')) DISCARDER_NUM_BUILDS = '1'
+if (!binding.hasVariable('DISCARDER_NUM_ARTIFACTS')) DISCARDER_NUM_ARTIFACTS = '1'
+
+def GIT_URI = 'https://github.com/samuel-rubin/openj9.git'
+def GIT_BRANCH = 'refs/heads/jenkins_automatic_pipeline'
+
+def pipelineScript = ''
+if (jobType == 'build'){
+    pipelineScript = 'buildenv/jenkins/jobs/builds/Build-Test-Any-Platform'
+} else if (jobType == 'pipeline'){
+    pipelineScript = 'buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-Any-Platform'
+}
+
+pipelineJob("$JOB_NAME") {
+    description('<h1>THIS IS AN AUTOMATICALLY GENERATED JOB DO NOT MODIFY, IT WILL BE OVERWRITTEN.</h1><p>This job is defined in Pipeline_Template.groovy in the openj9 repo, if you wish to change it modify that</p>')
+    definition {
+        cpsScm {
+            scm {
+                git {
+                    remote {
+                        url(GIT_URI)
+                    }
+                    branch("${GIT_BRANCH}")
+                    extensions {
+                        cleanBeforeCheckout()
+                    }
+                }
+            }
+            scriptPath(pipelineScript)
+            lightweight(true)
+        }
+    }
+    logRotator {
+        numToKeep(DISCARDER_NUM_BUILDS.toInteger())
+        artifactNumToKeep(DISCARDER_NUM_ARTIFACTS.toInteger())
+    }
+    parameters {
+        choiceParam('SDK_VERSION', ["${SDK_VERSION}"])
+        choiceParam('PLATFORM', ["${PLATFORM}"])
+        stringParam('OPENJDK_REPO')
+        stringParam('OPENJDK_BRANCH')
+        stringParam('OPENJDK_SHA')
+        stringParam('OPENJ9_REPO')
+        stringParam('OPENJ9_BRANCH')
+        stringParam('OPENJ9_SHA')
+        stringParam('OMR_REPO')
+        stringParam('OMR_BRANCH')
+        stringParam('OMR_SHA')
+        stringParam('ADOPTOPENJDK_REPO')
+        stringParam('ADOPTOPENJDK_BRANCH')
+        stringParam('VARIABLE_FILE', VARIABLE_FILE_DEFAULT)
+        stringParam('VENDOR_REPO', VENDOR_REPO_DEFAULT)
+        stringParam('VENDOR_BRANCH', VENDOR_BRANCH_DEFAULT)
+        stringParam('VENDOR_CREDENTIALS_ID', VENDOR_CREDENTIALS_ID_DEFAULT)
+        stringParam('SETUP_LABEL')
+        stringParam('BUILD_IDENTIFIER')
+        stringParam('ghprbGhRepository')
+        stringParam('ghprbActualCommit')
+        stringParam('EXTRA_GETSOURCE_OPTIONS')
+        stringParam('EXTRA_CONFIGURE_OPTIONS')
+        stringParam('EXTRA_MAKE_OPTIONS')
+        stringParam('OPENJDK_CLONE_DIR')
+        stringParam('PERSONAL_BUILD')
+        
+        if (jobType == 'pipeline'){
+            stringParam('TESTS_TARGETS')
+            stringParam('BUILD_NODE')
+            stringParam('TEST_NODE')
+            stringParam('SLACK_CHANNEL')
+            stringParam('RESTART_TIMEOUT')
+            stringParam('RESTART_TIMEOUT_UNITS')
+            stringParam('BUILD_LIST')
+            booleanParam('AUTOMATIC_GENERATION', true, 'uncheck to disable the automatic generation of jenkins jobs')
```

Comment:
I'd lean towards not having this description here since none of the others have it. The descriptions would be most useful in the wrapper jobs, which will be gen'd under a different task.